pyproject.toml

```diff
@@ -166,7 +166,7 @@ extra-dependencies = [
 ]
 
 [tool.hatch.envs.test.scripts]
-test = "pytest -n auto {args:tests}"
+test = "pytest {args:tests}"
```

Comment:
Note to reviewer: I discovered that `-n auto` makes running individual tests slower, so removing by default (can still be passed in)